### PR TITLE
Fixes cumulus README instructions

### PR DESCRIPTION
**Pull request by gpestana**
README instructions fixes to be compatible with the `polkadot-prepare` and `polkadot-execute` binary split.

### CODE REVIEW

File: cumulus/README.md [hunk 1]
```diff
@@ -161,26 +165,26 @@ cargo build --release --bin polkadot
 # Clone
 git clone https://github.com/paritytech/polkadot-sdk
 
-# Compile
+# Compile (under `/polkadot-sdk/cumulus/polkadot-parachain`)
 cargo build --release --bin polkadot-parachain
```
Comment (bkontur) [inline cumulus/README.md:169]:
```suggestion
cargo build --release -p polkadot-parachain-bin
```

File: cumulus/README.md [hunk 2]
```diff
@@ -161,26 +165,26 @@ cargo build --release --bin polkadot
 # Clone
 git clone https://github.com/paritytech/polkadot-sdk
 
-# Compile
+# Compile (under `/polkadot-sdk/cumulus/polkadot-parachain`)
```
Comment (bkontur) [inline cumulus/README.md:168]: I would remove this and better add there:
```
cd ./cumulus/polkadot-parachain`
cargo build --release --bin polkadot-parachain
```

or just use from the top directory:
```
cargo build --release -p polkadot-parachain-bin
```

File: cumulus/README.md (hunk 1 above)
Comment (bkchr) [reply cumulus/README.md:169]: Yeah this is much better!

File: cumulus/README.md [hunk 3]
```diff
@@ -145,6 +145,10 @@ git clone https://github.com/paritytech/polkadot-sdk
 # Compile Polkadot
 cargo build --release --bin polkadot
 
+# Compile Polkadot's `execute` and `prepare` binaries
+cargo build --release --bin polkadot-prepare
+cargo build --release --bin polkadot-execute
```
Comment (mrcnski) [inline cumulus/README.md:150]: Do we need the workers here? I thought they are only needed for validators. BTW, this looks a bit nicer (the parentheses start a subshell so we don't have to `cd ..` after):

```suggestion
(cd polkadot; cargo build --release)
```

Comment (bkchr) [reply cumulus/README.md:150]: Yes, because you need to onboard your parachain somewhere.

**Comment by bkchr** [comment]
@gpestana can you please fix the comments

File: cumulus/README.md (hunk 2 above)
Comment (gpestana) [reply cumulus/README.md:168]: Changed, thanks

**Pull request merged**